hi-ml-cpath/src/health_cpath/configs/classification/BaseMIL.py

```diff
@@ -73,6 +73,10 @@ class BaseMIL(LightningContainer, EncoderParams, PoolingParams):
                                                             "maximised (otherwise minimised).")
     ssl_checkpoint_run_id: str = param.String(default="", doc="Optional run id from which to load checkpoint if "
                                               "using SSLEncoder")
+    max_num_workers: int = param.Integer(10, bounds=(0, None),
```

Comment:
10 might be high as a default? what about 6? A lot of us faced some issues with num_workers and I think 6 was the limit that didn't give much headache.

Reply:
Cloud runs mostly, locally I can't go higher than 2

Reply:
as discussed, will stick with 10 for now, as it has worked fine on the large VMs.